fhir-server/pom.xml

```diff
@@ -121,15 +121,14 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>javax.inject</groupId>
-            <artifactId>javax.inject</artifactId>
-            <version>1</version>
+            <groupId>jakarta.inject</groupId>
+            <artifactId>jakarta.inject-api</artifactId>
+            <version>1.0.3</version>
```

Comment:
should this version be moved to dependencyManagement too?

Reply:
Yes

Reply:
Yes! I'm not sure how I missed it.